### DISCUSSION TITLE
assertequals javascript error expected undefined to equal 17 while log 17

### DISCUSSION THREAD

**Discussion by udha19** [codewars/codewars.com#3173] [Q&A]
I got this error, but I am not sure what is wrong, cause log and expected already match, isn't it?

![image](https://github.com/codewars/codewars.com/assets/12099999/36355ca3-ce8e-4ed2-bfdd-501c987d39c9)

**Comment by Blind4Basics** [comment]
This is most likely a kata issue: the arguments are swapped. Please open an issue in the kata discourse.

**Reply by udha19** [reply]
actually, its my bad, I use `console.log` instead of `return` for the result. 😅
